### ISSUE TITLE
[FE] - Update content for travel questions pages in travel-claim app

### ISSUE THREAD

**Issue by brianseek** [department-of-veterans-affairs/va.gov-team#75748]
## Description
Look for any content changes across the travel questions pages from check-in to travel-claim and make the updates in the travel-claim app. 

This includes:
 - travel-mileage
 - travel-vehicle
 - travel-address
 - travel-review

## Wireframes
- [Wireframes](https://www.figma.com/file/pnR05o7NPJDS0KFUSQ0eE3/Check-in-%7C-Check-in?type=design&node-id=96-709&mode=design&t=XEveKExW34rRJ9Z6-0)

## Tasks
- [ ] Update travel question page content
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations (readme and use cases)

## Acceptance Criteria
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Tested on staging
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature
- [ ] Prod validation:
  - [ ] not necessary
  - [ ] scheduled

## Code Review Steps
- [ ] Read ticket details and verify that ACs are met
- [ ] Review code in PR
- [ ] Run branch locally
- [ ] Check that new and updated translations use new keys
- [ ] Check for hardcoded strings

**Comment by brianseek** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Check-in-Experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/75748?planning-poker) with Zenhub @brianseek @lee-delarm6 @wullaski

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-vehicle

The accordion content should be: https://www.figma.com/file/pnR05o7NPJDS0KFUSQ0eE3/LoROTA-Travel-Reimbursement-%7C-Check-in?type=design&node-id=507-6853&mode=design&t=WHUPplgQZmiDWaQq-4

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-address

What's that "Answer "Yes"..." text styled as? Intro? We've made a recent change to stop using Intro. Change it to body, please.

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-address

Accordion content should be: https://www.figma.com/file/pnR05o7NPJDS0KFUSQ0eE3/LoROTA-Travel-Reimbursement-%7C-Check-in?type=design&node-id=507-8049&mode=design&t=WHUPplgQZmiDWaQq-4

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-review

What's that "You can submit..." paragraphs text styled as? Intro? We've made a recent change to stop using Intro. Change it to body, please.

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-review

"Mileage-only reimbursement..." with that hyphen underneath the What you're claiming heading AND remove period at the end of sentence. Not sure if that's hardcoded, programmatic, coming from the mock station value, etc.

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-review

Use fa-angle-double-right and left in buttons

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-agreement

Link at bottom should be styled as strong type link

**Comment by benbrasso-agile6** [comment]
I'm actually thinking that the "Edit" link on http://localhost:3001/my-health/appointment-travel-claim/travel-review

Should go back to the Intro page http://localhost:3001/my-health/appointment-travel-claim/travel-pay given that our back button and back to last screen breadcrumb sort of create a weird loop if Edit directs to the http://localhost:3001/my-health/appointment-travel-claim/travel-mileage page. 

This one's hard to follow if you're not going through it, but any thoughts from @brianseek @lee-delarm6 @cgednalske

**Comment by benbrasso-agile6** [comment]
On http://localhost:3001/my-health/appointment-travel-claim/travel-review

Do we need to include "By submitting..." paragraph in the FE validation error, i.e., no check on checkbox? For reference: https://www.figma.com/file/pnR05o7NPJDS0KFUSQ0eE3/LoROTA-Travel-Reimbursement-%7C-Check-in?type=design&node-id=1609-6521&mode=design&t=WHUPplgQZmiDWaQq-4

**Comment by brianseek** [comment]
@lee-delarm6 I'm going to take @benbrasso-agile6 's comments and fix them in my PR. We can close this for now.

**Comment by cgednalske** [comment]
Re edit link: Edit pattern is to edit the info they selected, so wouldn't we want to take them back to their info? On regular forms this would take them to the information they've already entered, not to before they started the form.

**Comment by benbrasso-agile6** [comment]
Yes, but when we do that, and you go back again, you go back to your last visited page, which is the Review page. It felt weird to me. Brian and I were both indifferent but kind of sided on the edit destination going back to the Intro. You feel strongly it should go back to the select appt page?

**Comment by cgednalske** [comment]
Ya that's an expected behavior for forms that going back to edit would go to the answers you need to edit (with answers saved), not start you off before your answers. 
Definitely see how the back button is weird, but is there a way to deal with the back button weirdness without compromising the expected behavior for the pattern?

**Comment by cgednalske** [comment]
Actually, isn't the back button weird regardless?

**Comment by benbrasso-agile6** [comment]
Plus, we can't really "save" someone's answers because there's no form fields 🙃 So, we'll just make best judgments tomorrow.
